tests/generation/SettingsTest.py

```diff
@@ -51,7 +51,7 @@ def test_write_info(self):
 
     def test_allow_complete_section_mode_with_ignore_globs(self):
         project_dir = "/repo"
-        project_files = ['/repo/hello.html']
+        project_files = ['/repo/hello.html', '/repo/readme.txt']
```

Comment:
https://github.com/coala/coala-quickstart/issues/279#issuecomment-436561208 I don't see any change in coverage on removal of that regex. Can you explain the reason for this change?

Reply:
If the code `, '/repo/readme.txt'` is removed and pytest is run, coverage is not 100%. Following is the screenshot for it:
![ss](https://user-images.githubusercontent.com/15020747/48130238-ff976200-e2b1-11e8-8345-a11ba573dba7.png)

This is becuase `if ext in exts:` in Utilities.py does not have 100% coverage.